db/controllers/articles.controller.js

```diff
@@ -24,3 +24,21 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (typeof inc_votes !== "number") {
+    return res.status(400).send({ msg: "Bad request - invalid input" });
+  }
+
+  selectArticleById(article_id).then(() => {
```

Comment:
not necessary to select first - article_id is a foreign key so if we try to update and it doesn't find the article by the id it will error!